app/javascript/gobierto_dashboards/modules/contracts_controller.js

```diff
@@ -184,16 +184,14 @@ export class ContractsController {
   }
 
   _renderContractsMetricsBox(){
-    const _contractsData = this._currentDataSource().contractsData
+    const _contractsData = this._formalizedContracts();
+
     // Calculations
-    const amountsArray = _contractsData.map(({final_amount = 0}) => final_amount === '' ? 0.0 : parseFloat(final_amount) );
+    const amountsArray = _contractsData.map(({final_amount = 0}) => parseFloat(final_amount) );
     const sortedAmountsArray = amountsArray.sort((a, b) => b - a);
-    const savingsArray = _contractsData.map(({initial_amount = 0, final_amount = 0}) =>{
-      initial_amount = initial_amount === '' ? 0.0 : initial_amount;
-      final_amount = final_amount === '' ? 0.0 : final_amount;
-
-      return (1 - parseFloat(final_amount) / parseFloat(initial_amount))
-    });
+    const savingsArray = _contractsData.map(({initial_amount = 0, final_amount = 0}) =>
+      1 - parseFloat(final_amount) / parseFloat(initial_amount)
```

Comment:
Is this your operation, right?
```suggestion
      1 - (parseFloat(final_amount) / parseFloat(initial_amount))
```